src/bit_str.jl

```diff
@@ -288,6 +288,9 @@ Returns an onehot vector of type `Vector{T}`, where the `bit_str`-th element is
 onehot(::Type{T}, n::BitStr) where T = onehot(T, length(n), n.val)
 onehot(n::BitStr) = onehot(Float64, n)
 
+onehot_batch(::Type{T}, n::BitStr, nbatch::Int) where T = onehot_batch(T, length(n), n.val, nbatch)
```

Comment:
Why not using the multiple dispatch instead of a new function name for this?

Reply:
you mean use

```julia
onehot(::Type{T}, n::BitStr, nbatch::Int)
```

instead? I'm good with this.

Reply:
yes, it is better